src/components/ProposalActions.js

```diff
@@ -143,20 +134,26 @@ function ProposalActions({
             `}
           >
             <Slider
-              value={progress}
-              onUpdate={setProgress}
+              value={
+                maxAvailable.gt(0) ? amount.div(maxAvailable).toNumber() : 0
+              }
+              onUpdate={newProgress =>
```

Comment:
Let's memoize this functions, with `useCallback`, otherwise they'll be re created on every re-render.

Reply:
Done.